kolibri/plugins/facility_management/assets/src/views/UserTable.vue

```diff
@@ -177,7 +179,15 @@
   }
 
   .role-badge {
+    display: inline-block;
+    padding-right: 1em;
+    padding-left: 1em;
     margin-left: 8px;
+    font-size: small;
+    color: $core-bg-light;
+    white-space: nowrap;
+    background-color: $core-text-annotation;
+    border-radius: 0.5em;
```

Comment:
Let's increase the line-height and make to make the pill thiccer.

Reply:
Not specced!!